src/content/protocols/sky.md

```diff
@@ -0,0 +1,381 @@
+
+# Overview
+
+## Chain
+
+This review focuses on the Ethereum deployment of Sky, currently the only chain the Sky protocol is deployed on. As part of Sky's _End Game_ plan the protocol is gradually deploying on other chains and aims to launch its own chain in the future.
+
+> Chain score: Low
+
+## Upgradeability
+
+`USDS` and `sUSDS` are upgradeable contracts through governance proposals. Updating those contracts could change the entire logic of those tokens and may incur loss of funds for users.
+
+Many parameters in the Sky Protocol can be changed through governance proposals and may also incur loss of funds, loss of unclaimed yield, as well as material changes in expected performances. Example actions are forced liquidations, creating unbacked debt, or pausing the contracts.
+
+> Upgradeability score: High
+
+## Autonomy
+
+The Sky protocol relies on the provider Chronicle for price feeds of collateral assets. The Chronicol protocol uses a system of decentralized validators (and challengers) to push price updates on-chain and is explained in more details in [dependencies](#dependencies).
+
+An Oracle Security Module (`OSM`) enforces a 1-hour delay on price updates and the governance can freeze the current price to prevent further updates in case of emergency. The price feed provider could be changed through a governance proposal.
+
+> Autonomy score: Medium
+
+## Exit Window
+
+All permissions within Sky are held by the onchain Sky Governance. There are no external accounts or multisigs in control.
+
+Governance is controlled by `MKR`/`SKY` token holders who locked their tokens to earn voting power which allows them to submit and vote on respective proposals, or delegate their votes to another user. Any token holder can create a proposal and the protocol uses a system of continuous approval explained in more detail in the [technical section](#exit-window-1).
+
+The minimum delay between approval and execution of a governance proposal is currently **18 hours**. Governance proposals have a recurring weekly and monthly schedules. Those proposals usually have a courtesy delay duration of 2-3 days before taking effect, but proposals remain possible at any time with the minimum delay.
+
+Emergency measures allow the governance to pause certain contracts through a governance proposal without being subject to the mandatory delay. In addition to that, an _Emergency Shutdown Module_ (ESM) exists and can irreversibly shutdown the entire protocol if **500'000** `MKR` tokens are sent to the Emergency Shutdown Contract. Funds sent to the contract are considered lost no matter the emergency status.
```

Comment:
maybe mention that for the protocol shutdown, users are able to withdraw their supplied assets but not supply or borrow more